src/document/codefmt.ml

```diff
@@ -180,7 +210,11 @@ let rec list ?sep ~f = function
       let tl = list ?sep ~f xs in
       match sep with None -> hd ++ tl | Some sep -> hd ++ sep ++ tl)
 
-let render f = spf "@[%t@]" (span f)
+let box_hv t ppf = pf ppf "@[<hv 2>%t@]" t
```

Comment:
This API is getting bigger, it's time to make it more abstract: https://github.com/panglesd/odoc/pull/1 (a PR to your PR, feel free to squash)

Reply:
Oops I forgot to squash. I can rewrite history if you prefer.

Reply:
No need to squash unless you want to :)